.circleci/config.yml

```diff
@@ -10,15 +10,19 @@ jobs:
           command: |
             sudo pip install --upgrade pre-commit==2.15.0
             pre-commit run --all-files
+  test-and-build:
+    docker:
+      - image: circleci/python:3.9-buster
+    steps:
+      - checkout
       - setup_remote_docker:
-          version: 17.10.0-ce
+          version: 20.10.7
```

Comment:
That was needed :)
Can we actually bump docker in our current setup - before merging this?